code/modules/mob/living/devour.dm

```diff
@@ -44,6 +46,16 @@
 		src << "\red You can't eat yourself!"
 		return 0
 
+	if (devouring == victim)
```

Comment:
I wonder if it'd be better to have `if (devouring)` here as well. It's technically in the for() loop, but it might be better in terms of feedback to put it here with a return instead.

```
if (devouring)
    src << span("danger", "You're already eating [devouring]!")
    return
```

Otherwise, there's no explicit feedback on why the devouring was cancelled if you swap targets.